_topic_map.yml

```diff
@@ -117,21 +119,21 @@ Topics:
 - Name: Managing administration roles and users
   File: osd-admin-roles
 ---
-Name: Setting up clusters and accounts using AWS security token service (STS)
-Dir: rosa_getting_started
+Name: Setting up accounts and clusters using AWS security token service (STS)
+Dir: rosa_getting_started_sts
 Distros: openshift-rosa
 Topics:
 - Name: Getting started using STS workflow
```

Comment:
Good questions and comments. Quick answers:

Yes, we need the word `workflow`. Otherwise, customers might assume that it's just an overview topic instead of a valuable high-level process topic. This was a huge customer improvement that was well-received by SRE too. 

PrivateLink should not be added to the heading, because this heading covers a regular ROSA provisioning workflow that offers the option of creating multiple cluster types, including PrivateLink clusters.

Because of the maintenance burden in Asciidoc, and the overall similarity of this type of the regular ROSA/privatecluster/PrivateLink cluster provisioning, it's best to keep these as one type of provisioning (the other provisioning type being STS).

Reply:
I understand your point but we don't use the word workflow for Getting Started in other OCP docs. I'm not going to insist because of this PR's urgency, but let's talk later.